components/library/ncurses/Makefile

```diff
@@ -61,31 +59,26 @@ CFLAGS += -std=c99 -D_POSIX_C_SOURCE=200112L
 
 CONFIGURE_OPTIONS +=	--with-shared
 CONFIGURE_OPTIONS +=	--with-normal
+CONFIGURE_OPTIONS +=	--with-abi-version=5
 CONFIGURE_OPTIONS +=	--enable-rpath
 CONFIGURE_OPTIONS +=	--enable-widec
-CONFIGURE_OPTIONS +=	--enable-pc-files
+CONFIGURE_OPTIONS +=	--enable-symlinks
+CONFIGURE_OPTIONS +=	--disable-lib-suffixes
 CONFIGURE_OPTIONS +=	--includedir=$(USRINCDIR)/ncurses
+CONFIGURE_OPTIONS +=	--with-pkg-config-libdir=$(PKG_CONFIG_PATH)
+CONFIGURE_OPTIONS +=	--enable-pc-files
 
 CONFIGURE_ENV += CPPFLAGS="$(CPPFLAGS)"
 
-# XXX: Ugly hack copied directly from the desktop SUNWncurses.spec file:
```

Comment:
Also need to restore this hack

Reply:
With or without it after updating ncurses text-installer draws '?' instead of window boundaries in console.

Reply:
Likely, we have to wait for console update in illumos-gate to happen before integrating this update.